rocketbelt/base/typography/_typography-cta.scss

```diff
@@ -1,10 +1,12 @@
 // Color is going to change.
-.cta {
+.cta,
+a.cta {
```

Comment:
Not sure we need extra specificity in this case—can you please remove this line? If we come up with edge cases that require an additional selector we can add them in the future.